### PR TITLE
remove col-2 col-3 and added bootstrap grid template

### PR DESCRIPTION
**Pull request by herecomesjaycee**
@womenhackfornonprofits/whfnp
## What are the changes included in this PR?

I removed the col-2 and col-3 and put in bootstrap grid template.

### CODE REVIEW

**Comment by lili2311** [comment]
Could you please do a screenshot for desktop and mobile so it is easier to say if the layout os good to go or needs some tweaking before we merge? Otherwise 👍

**Comment by herecomesjaycee** [comment]
screen shot 2016-09-05 at 22 28 33
screen shot 2016-09-05 at 22 28 38
screen shot 2016-09-05 at 22 28 40
There is an issue with the latest Gruntfile.js where I can't run Grunt >> SyntaxError: Unexpected token <<

**Comment by lili2311** [comment]
Thanks! Strange you don't have any images? Is grunt task running? 

So the speech bubbles look too big on desktop I think they need to be maybe a col-4 or even 3 with an offset so that they both stay in the center of the page with their edges if that makes sense?

**Comment by herecomesjaycee** [comment]
There is an issue with the latest Gruntfile.js where I can't run Grunt ">> SyntaxError: Unexpected token <<" and it is the latest version of the file on Github. Shall make the change on the boostrap :)

**Comment by lili2311** [comment]
Sounds like there was a merge conflict maybe if you post your gruntfile we can have you push a fix for it

**Comment by herecomesjaycee** [comment]
Oh dear! How may I post my gruntfile?

**Comment by herecomesjaycee** [comment]
screen shot 2016-09-18 at 14 55 17 - Mobile view 
screen shot 2016-09-18 at 14 54 56 - Desktop view

File: src/non-profits.html [hunk 1]
```diff
@@ -78,28 +78,28 @@ <h2 class="section-heading">Our Clients</h2>
     <p>
       <div class="non-profits-wrap">
         <div class="client-testimonial">
-          <div class="col-2">
+          <div class="col-xs-6">
             <img src="img/non-profits/jaspals-voice-logo.png" alt="Jaspal's Voice Logo Image">
           </div>
-          <div class="col-2 speech-bubble-left">
+          <div class="col-xs-6speech-bubble-left">
```
Comment (lili2311) [inline src/non-profits.html]: Looks like there is a space missing here between the classes?

Comment (lili2311) [reply src/non-profits.html]: other than that looks great :)

Comment (herecomesjaycee) [reply src/non-profits.html]: @lili2311 Thanks for the review, that is rectified now and let me know if you are happy for me to merge.

**Review by lili2311 — APPROVED** [review]

**Pull request merged**